addons/sourcemod/scripting/l4d2_lagcomp_manager.sp

```diff
@@ -0,0 +1,56 @@
+#include <sourcemod>
+#include <sdktools>
+
+public Plugin myinfo =
+{
+	name = "L4D2 Lag Compensation Manager",
+	author = "ProdigySim",
+	description = "Provides lag compensation for stuff in left four dead two",
+	version = "One",
+	url = "https://github.com/SirPlease/L4D2-Competitive-Rework"
+};
+
+Address g_lagcompensation = Address_Null;
+Handle g_hLagCompAddEntity = null;
+Handle g_hLagCompRemoveEntity = null;
+
+public void OnPluginStart()
+{
+	Handle hGameConf = LoadGameConfigFile("l4d2_lagcomp_manager");
+
+	g_lagcompensation = GameConfGetAddress(hGameConf, "lagcompensation");
+	StartPrepSDKCall(SDKCall_Raw);
+	PrepSDKCall_SetFromConf(hGameConf, SDKConf_Signature, "CLagCompensationManager_AddAdditionaEntity");
+	PrepSDKCall_AddParameter(SDKType_CBaseEntity, SDKPass_Pointer);
+	g_hLagCompAddEntity = EndPrepSDKCall();
+
+	StartPrepSDKCall(SDKCall_Raw);
+	PrepSDKCall_SetFromConf(hGameConf, SDKConf_Signature, "CLagCompensationManager_RemoveAdditionaEntity");
+	PrepSDKCall_AddParameter(SDKType_CBaseEntity, SDKPass_Pointer);
+	g_hLagCompRemoveEntity = EndPrepSDKCall();
+	// PrintToServer("Values: 0x%08x, %08x, %08x", g_lagcompensation, g_hLagCompAddEntity, g_hLagCompRemoveEntity);
```

Comment:
TODO: Add a SetFailState or similar so we don't have to go searching if these signatures fail?